recipe/build.sh

```diff
@@ -20,6 +20,7 @@ export PYARROW_WITH_ORC=1
 export PYARROW_WITH_PARQUET=1
 export PYARROW_WITH_PLASMA=1
 export PYARROW_WITH_S3=1
+export PYARROW_CMAKE_OPTIONS="-DARROW_ARMV8_ARCH=armv8-a"
```

Comment:
Is this correct? `armv8` normally stands for the 32bit variant, we only build for `aarch64`.

Reply:
Well, I set it to the [default](https://github.com/apache/arrow/blob/master/cpp/cmake_modules/DefineOptions.cmake#L113). Seems like didn't need to set it in arrow-cpp-feedstock, so I guess the default value is used in those builds.